src/Keboola/DbExtractor/RetryProxy.php

```diff
@@ -88,7 +88,8 @@ public function call(callable $action, array $arguments = [])
             }
         } while ($this->retryPolicy->canRetry($retryContext));
 
-        if ($lastException = $retryContext->getLastException()) {
+        $lastException = $retryContext->getLastException();
+        if ($lastException instanceof \Exception) {
```

Comment:
\Throwable  ?

Reply:
asi nějaký úlet - opraveno

Reply:
no, ne, mas tam porad \Exception a melo by tam byt Throwable - https://www.php.net/manual/en/language.errors.php7.php . Mel jsem pocit, ze to je v CS rules

Reply:
ano, \Exception does not catch ErrorException

Reply:
omg - je to tam